config/environments/remote_logger.rb

```diff
@@ -12,5 +12,4 @@
 
   config.log_formatter = ::Logger::Formatter.new
   config.log_level = (Figaro.env.REMOTE_LOG_LEVEL || 'info').to_sym
-  config.logger = logger
```

Comment:
Removed this line because the line higher up which isn't visible goes directly to `config.logger` so `logger` is never set and this line is unnecessary.